src/trading/Orders.sol

```diff
@@ -0,0 +1,320 @@
+        return bestOrder[getBestOrderWorstOrderHash(_market, _outcome, _type)];
+    }
+
+    function getWorstOrders(Trading.TradeTypes _type, Market _market, uint8 _outcome) public constant returns (bytes20) {
+        return worstOrder[getBestOrderWorstOrderHash(_market, _outcome, _type)];
+    }
+
+    function getAmount(bytes20 _orderId, Trading.TradeTypes _type, Market _market, uint8 _outcome) public constant returns (uint256) {
+        return orders[_orderId].fxpAmount;
+    }
+
+    function getPrice(bytes20 _orderId, Trading.TradeTypes _type, Market _market, uint8 _outcome) public constant returns (int256) {
+        return orders[_orderId].fxpPrice;
+    }
+
+    function getOrderOwner(bytes20 _orderId, Trading.TradeTypes _type, Market _market, uint8 _outcome) public constant returns (address) {
+        return orders[_orderId].owner;
+    }
+
+    function getOrderSharesEscrowed(bytes20 _orderId, Trading.TradeTypes _type, Market _market, uint8 _outcome) public constant returns (uint256) {
+        return orders[_orderId].fxpSharesEscrowed;
+    }
+
+    function getOrderMoneyEscrowed(bytes20 _orderId, Trading.TradeTypes _type, Market _market, uint8 _outcome) public constant returns (uint256) {
+        return orders[_orderId].fxpMoneyEscrowed;
+    }
+
+    function getVolume(Market _market) public constant returns (uint256) {
+        return marketOrderData[_market].volume;
+    }
+
+    function getLastOutcomePrice(Market _market, uint8 _outcome) public constant returns (int256) {
+        return marketOrderData[_market].prices[_outcome];
+    }
+
+    function getBetterOrderId(bytes20 _orderId, Trading.TradeTypes _type, Market _market, uint8 _outcome) public constant returns (bytes20) {
+        return orders[_orderId].betterOrderId;
+    }
+
+    function getWorseOrderId(bytes20 _orderId, Trading.TradeTypes _type, Market _market, uint8 _outcome) public constant returns (bytes20) {
+        return orders[_orderId].worseOrderId;
+    }
+
+    function getBestOrderId(Trading.TradeTypes _type, Market _market, uint8 _outcome) public constant returns (bytes20) {
+        return bestOrder[getBestOrderWorstOrderHash(_market, _outcome, _type)];
+    }
+
+    function getWorstOrderId(Trading.TradeTypes _type, Market _market, uint8 _outcome) public constant returns (bytes20) {
+        return worstOrder[getBestOrderWorstOrderHash(_market, _outcome, _type)];
+    }
+
+    // FIXME: Currently, getOrderIdHash() & getBestOrderWorstOrderHash() are creating ripemd160 hash values of type bytes20.  This should probably be changed to create sha3/sha256 hash values of type bytes32.
+    function getOrderIdHash(Trading.TradeTypes _type, Market _market, uint256 _fxpAmount, int256 _fxpPrice, address _sender, uint256 _blockNumber, uint8 _outcome, uint256 _fxpMoneyEscrowed, uint256 _fxpSharesEscrowed) public constant returns (bytes20) {
```

Comment:
This should be renamed to `getOrderId`.  The fact that it happens to be a hash is an implementation detail.